src/core/CMakeLists.txt

```diff
@@ -342,7 +350,8 @@ target_link_libraries(
          GDAL::GDAL
          SQLite::SQLite3
          Qca::qca
-         libzip::zip)
+         libzip::zip
+         GSL::gsl)
```

Comment:
This is a transitive dependency, can you add it through cmake/qgis-cmake-wrapper.cmake (QGIS::Core? QGIS::Analysis?)

Reply:
done.